examples/tutorials/advanced/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,1112 @@
+{
```

Comment:
<p>Q: With the PySyft Projects should we better point here "https://github.com/OpenMined"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Yes I agree, changed that. </p>